phpunit.xml.dist

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" colors="true" stderr="true" convertErrorsToExceptions="true" convertNoticesToExceptions="true" convertWarningsToExceptions="true" stopOnFailure="false" verbose="true" bootstrap="tests/bootstrap.php" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
-  <coverage>
-    <include>
+  <filter>
+    <whitelist>
       <directory suffix=".php">./src</directory>
-    </include>
-  </coverage>
+    </whitelist>
+  </filter>
```

Comment:
Similarly this is to fix the error for coverage

<details>
<summary>Error: coverage: element is not expected</summary>

<img width="551" alt="Screenshot 2023-04-12 at 2 02 35 PM" src="https://user-images.githubusercontent.com/8046636/231419811-3da53ecd-6811-4225-81a7-22799b2fef73.png">

</details>